### DISCUSSION TITLE
integration of switch drivers, recuperation, etc.?

### DISCUSSION THREAD

**Discussion by DavidK978** [JoDehli/PyLoxone#303] [Ideas]
hello, everything works great, but there are some things missing, but you are a great bunch that does it right! 

would it be possible to add these things please? Thank you very much. I think it would help other people too. 

switch and controller: https://shop.loxone.com/cscz/touch-for-nano-bila.html would be great to program pink actions on this button would it be possible to add it? I have several and unfortunately I don't see it in Pyloxone. 

It would also be great if it would be possible to change the speed of the recuperation/ventilation via PyLoxone. or switch on the underfloor heating.  Thank you so much for what you are doing and I will look forward to updates. :)

**Comment by JoDehli** [comment]
@DavidK978 how many touch entities do you have? I think with manual work you can get it to work with the current implementation. Problem with special devices from loxone is that we do not have a equal entity on ha. For 4 or 5 functions we could generate 4 or five switches and group it together.
